test/unittests/eof_validation_test.cpp

```diff
@@ -193,14 +181,6 @@ TEST_F(eof_validation, EOF1_truncated_section)
     add_test_case("EF0001 010004 0200010002 040000 00 00800000 FE",
         EOFValidationError::invalid_section_bodies_size);
 
-    // Data section may be truncated in runtime subcontainer
-    add_test_case(
-        eof_bytecode(returncontract(0, 0, 2), 2).container(eof_bytecode(OP_INVALID).data("", 2)),
-        ContainerKind::initcode, EOFValidationError::success);
-    add_test_case(
-        eof_bytecode(returncontract(0, 0, 1), 2).container(eof_bytecode(OP_INVALID).data("aa", 2)),
```

Comment:
I can't find where did this go?

Reply:
[EOF1_embedded_container](https://github.com/hugo-dc/evmone/blob/31b27d296071dbfe3eef93d7201916a081296348/test/unittests/eof_validation_test.cpp#L951-L954)